app/views/layouts/application.html.haml

```diff
@@ -26,7 +26,7 @@
     = javascript_include_tag 'application'
     %meta{content: asset_path("govuk-opengraph-image.png"), property: "og:image"}/
     = csrf_meta_tags
-  %body.govuk-template__body.app-body-class
+  %body{class: "govuk-template__body app-body-class #{authenticated? ? 'hiring-staff': '' } #{controller_name + '_' + action_name}"}
```

Comment:
We could move this logic into a tested helper method to keep the view simpler and to help us catch any regressions.